administrator/components/com_content/controllers/article.php

```diff
@@ -156,7 +156,9 @@ public function batch($model = null)
 	 */
 	protected function postSaveHook(JModelLegacy $model, $validData = array())
 	{
-
+		require_once JPATH_ADMINISTRATOR . '/components/com_menus/models/item.php';
+		$itemmodel = JModelAdmin::getInstance('Item', 'MenusModel', array());
+		$itemmodel->save($validData);
```

Comment:
Instead of sending whole, `$validData` which is actually valid for article, you should prepare new data array to send Menu model.